.github/workflows/build.yml

```diff
@@ -0,0 +1,86 @@
+name: Dolphin-AppImage
+
+on: 
+  push:
```

Comment:
```suggestion
on: push
```

Reply:
@qurious-pixel you can just press OK here and github automatically applies the suggested changes